src/MerchantCenter/MerchantStatuses.php

```diff
@@ -108,15 +108,18 @@ public function __construct() {
 	public function get_product_statistics( bool $force_refresh = false ): array {
 		$this->maybe_refresh_status_data( $force_refresh );
 
-		$counting_stats = $this->mc_statuses['statistics'];
-		$counting_stats = array_merge(
-			[ 'active' => $counting_stats[ MCStatus::PARTIALLY_APPROVED ] + $counting_stats[ MCStatus::APPROVED ] ],
-			$counting_stats
-		);
-		unset( $counting_stats[ MCStatus::PARTIALLY_APPROVED ], $counting_stats[ MCStatus::APPROVED ] );
+		$counting_stats = is_array( $this->mc_statuses ) ? $this->mc_statuses['statistics'] : null;
```

Comment:
This can still produce a warning if it's an array with `statistics` not set. `isset` would be a little safer to catch both.

```suggestion
		$counting_stats = isset( $this->mc_statuses['statistics'] ) ? $this->mc_statuses['statistics'] : null;
```